packages/app/src/cli/services/dev/processes/app-logs-polling.ts

```diff
@@ -46,7 +46,7 @@ export async function setupAppLogsPollingProcess({
 }
 
 export const subscribeAndStartPolling: DevProcessFunction<SubscribeAndStartPollingOptions> = async (
-  {stdout},
+  {stdout, stderr, abortSignal},
```

Comment:
The `DevProcessFunction` demands to have these available when this is called, so I'm passing them in here. We only use them to re-call this dev process function.